www/includes/easyparliament/templates/html/static/help.php

```diff
@@ -416,29 +393,32 @@
                     Parliament undertakes a rolling process of corrections and updates. When their content is amended, the
                     next time our site picks it up, our content should be amended too.
                 </p>
+	
+				<p>
+				So — if the error is in the data, it's likely to resolve itself. But if there's something up with our
+				code or the way the page is displaying, etc, we would be glad to know.
+				</p>
 
                 <h3 class="js-toc-item" id="new-feature">
                     I have an idea for a new feature &ndash; do you want to hear it?
                 </h3>
                 <p>
-                    We&rsquo;d love to &ndash; but please be aware that we&rsquo;re a small team, so we can&rsquo;t always
-                    introduce new features as quickly as we&rsquo;d like to. However:
+                    We&rsquo;d love to – but please be aware that TheyWorkForYou is currently running without dedicated funding or resources,
+					so at the moment we're unlikely to introduce new features. However, when we do have funding,we adopt new ideas on a merit
```

Comment:
space after comma missing

Reply:
I think this is resolved but not showing up in the preview, does it need a special character or literally just the space? https://github.com/mysociety/theyworkforyou/blob/twfy-faq-update/www/includes/easyparliament/templates/html/static/help.php

Reply:
I was referring to "funding,we" on this line, not the comma at the end of the line above it :)

Reply:
doh.